compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -290,11 +293,11 @@ enum ImplTraitPosition {
     ImplReturn,
 }
 
-impl<'a> ImplTraitContext<'_, 'a> {
-    fn reborrow<'this>(&'this mut self) -> ImplTraitContext<'this, 'a> {
+impl ImplTraitContext {
+    fn reborrow<'this>(&'this mut self) -> ImplTraitContext {
```

Comment:
is `'this` still relevant?

Reply:
Anyway the whole reborrow impl was removed in a later commit.